### PR TITLE
Make links.js support old browsers. Fixes #9

### LINKED ISSUE #9: links.js - IE support
`links.js` uses `Element.prototype.closest` which is [not supported in IE](https://developer.mozilla.org/pt-BR/docs/Web/API/Element/closest) and some older versions of major browsers.

Since it's a simple operation, can we replace it with something like this?

```js
function findClosest(tagName, el) {
	while(el && el.tagName !== tagName) el = el.parentNode
	return el
}
```

```js
function onClick(event) {
	const anchor = findClosest('A', event.target)
	...
```

### PR DESCRIPTION
**Pull request by kaisermann**

### CODE REVIEW

File: links.js [hunk 1]
```diff
@@ -24,8 +24,15 @@ export default function links(node) {
   }
 }
 
+function findClosest(tagName, el) {
+	while(el && el.tagName !== tagName) {
+    el = el.parentNode;
+  }
+	return el;
+}
+
```
Comment (EmilTholin) [inline links.js]: Awesome stuff! Indentation looks a bit odd.

**Comment by EmilTholin** [comment]
No problem!
What do you think about changing the while loop to `while(el && el.tagName !== tagName && !el.target)`, so that it doesn't pick an anchor tag with a target attribute, like it currently behaves?

**Comment by kaisermann** [comment]
I think that it would make more sense to add this check on the following `if`, because it's not possible to have an `<a>` inside another `<a>` (like `<a href="/test"><a target="..."><span>target</span></a></a>`), so there's no need to be on the `while` loop.

image

**Comment by EmilTholin** [comment]
That's a good point. That sounds much better.

**Pull request merged**